README.md

```diff
@@ -737,6 +737,34 @@ for family in text_string_to_metric_families(u"my_gauge 1.0\n"):
     print("Name: {0} Labels: {1} Value: {2}".format(*sample))
 ```
 
+## Restricted registry
+
+Registry support restriction to only return specific metrics.
+If you’re using the built-in HTTP server, you can use the GET parameter "name []", since it’s an array it can be used multiple times.
```

Comment:
```suggestion
If you’re using the built-in HTTP server, you can use the GET parameter "name[]", since it’s an array it can be used multiple times.
```